tests/Doctrine/Tests/DBAL/Schema/Visitor/CreateSchemaSqlCollectorTest.php

```diff
@@ -76,7 +76,7 @@ public function testAcceptsTable(): void
 
     public function testAcceptsForeignKeyDoesNotSupportCreateDropForeignKeyConstraints(): void
     {
-        $this->platformMock->method('supportsCreateDropForeignKeyConstraints')
+        $this->platformMock->method('supportsForeignKeyConstraints')
```

Comment:
This also warrants the test name change. Should have been done in `2.10.x` but let's at least fix it here.